Procfile

```diff
@@ -1,3 +1,4 @@
 web: bin/run
 worker: bash -c "set -e; while true ; do php bin/console messenger:consume async --memory-limit=256M --limit=50 ; done"
 postdeploy: make scalingo-postdeploy
+scheduler: node vendor/node_modules/.bin/node-cron
```

Comment:
En regardant sur cette doc https://doc.scalingo.com/platform/app/task-scheduling/scalingo-scheduler#enabling-the-scalingo-scheduler

Ça parle bien du fichier `cron.json` mais pas d'un container `scheduler`, et je ne vois pas non plus où on utilise le nouveau paquet `dragonmantank/cron-expression` ?

Si ça n'est pas le système de Scalingo, est-ce que ça ne risque pas de rentrer en conflit avec lui ? On pourrait plutôt utiliser le scheduling de Scalingo avec leur format de `cron.json` ?